apps/researcher/src/app/[locale]/communities/[id]/page.tsx

```diff
@@ -0,0 +1,107 @@
+import {ChevronLeftIcon} from '@heroicons/react/24/solid';
+import Link from 'next/link';
+import Image from 'next/image';
+import {getTranslations} from 'next-intl/server';
+import {JoinCommunityButton, EditCommunityButton} from './buttons';
+import {getMemberships, getCommunity, isAdmin} from '@/lib/community';
+import ErrorMessage from '@/components/error-message';
+
+interface Props {
+  params: {id: string};
+}
+
+interface Error {
+  status: number;
+}
```

Comment:
Sorry, I overlooked this one during my previous review.

You can define your own interface, but I'd say that the error that is returned by `getCommunity()` already has a definition for this? For example, in pseudo-code:

```
import type {SomeClerkError} from 'clerk';

try {
  community = await getCommunity(params.id);
} catch (err) {
  const error = err as SomeClerkError;
  if (error.status === 404) {
    ...
}
```

Reply:
Ah, too bad (and really strange -- they do not export their error definitions?)!

Two considerations:

1. Are we allowed to import `@clerk/shared` (e.g. in our `devDependencies`)? Or is package for internal use by Clerk only?
2. If (1) is not an option: perhaps we can rename the current `interface Error` to something descriptive, so as to prevent confusion with JavaScript's default `Error` and to make clear that we're defining an interface that should have come from Clerk? For example: `interface ClerkError` (+ a comment explaining the reason why this interface exists)?

Reply:
Solved by installing @clerk/shared as a dev dependency.